src/com/core/tictactoe/Board.java

```diff
@@ -75,6 +75,24 @@ String winnerSign() {
         return freePlaces;
     }
 
+    String getActivePlayerSign() {
+        int XPlayerSignsCounter = 0;
+        int OPlayerSignsCounter = 0;
+        for (String place : this.places) {
+            if(place.equals("X")) {
+                XPlayerSignsCounter++;
+            } else if (place.equals("O")){
+                OPlayerSignsCounter++;
+            }
+        }
+        return XPlayerSignsCounter > OPlayerSignsCounter ? "O" : "X";
+    }
+
+    String getPassivePlayerSign() {
+        String activePlayerSign = getActivePlayerSign();
+        return activePlayerSign.equals("X") ? "O" : "X";
+    }
+
```

Comment:
The code might be more maintainable if you used constants for "X" and "O" instead of strings where you use them.

Reply:
Done.